### PR TITLE
Ck2

### PR DESCRIPTION
**Pull request by ckarpowicz**

### CODE REVIEW

**Comment by ckarpowicz** [comment]
ck2 seedclass annotations

File: src/main/java/com/example/model/SeedClass.java [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
+import javax.persistence.*;
 import java.sql.Timestamp;
 
 /**
  * Created by qzh225 on 5/3/17.
  */
+
+
+@Entity
+@Table(name = "seedclass")
```
Comment (SteveMustanski) [inline src/main/java/com/example/model/SeedClass.java:12]: add seedhub. to the table name

File: src/main/java/com/example/model/SeedClass.java [hunk 2]
```diff
@@ -1,18 +1,41 @@
...
+    @Id
+    @SequenceGenerator(name = "seedclassSequence", sequenceName = "seedclass_c_id_seq", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seedclassSequence")
 
+    //@OneToOne(mappedBy = "application", fetch = FetchType.EAGER)\
+    @OneToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "c_id")
     private int cId;
+
+    @Column(name = "cYr")
```
Comment (SteveMustanski) [inline src/main/java/com/example/model/SeedClass.java:23]: match the database table in the @column(name="should be database format name"

File: src/main/java/com/example/model/SeedClass.java [hunk 3]
```diff
@@ -1,18 +1,41 @@
...
+@Entity
+@Table(name = "seedclass")
 public class SeedClass {
+    @Id
+    @SequenceGenerator(name = "seedclassSequence", sequenceName = "seedclass_c_id_seq", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seedclassSequence")
 
+    //@OneToOne(mappedBy = "application", fetch = FetchType.EAGER)\
+    @OneToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "c_id")
```
Comment (SteveMustanski) [inline src/main/java/com/example/model/SeedClass.java:20]: no joins going out on this table

**Review by SteveMustanski — CHANGES_REQUESTED** [review]
look at the comments...  good job though...

**Pull request closed without merging**